card-data/01-character-upbringing.yaml

```
@@ -0,0 +1,264 @@
+- front: &Insular
+    <<: *tfront
+    name: Insular
+    desc: >
+      //"This is all very different from the cloister" -- Emory
+
+      You grew up secluded from the rest of the world.
+      Whether you lived on a lonely mountain village,
+      a religious retreat, or in an enclave within a
+      city, the ways of the outside world are a mystery.
+
+      You've learned to approach situations through curiosity.
+    prompts:
+      - Solve a problem using your peoples' ways
+      - Ask illuminating questions
+      - Reveal hidden knowledge or exclusive experience
+  back:
+    <<: *tback
+    name: Insular
+    desc: >
+      //"Everything's different out here. That's why it's called a cloister" -- Basler
+
+      Your early life didn't prepare you for the sheer
+      variey of people and things the world has to offer.
+      You might make a basic blunder, or ask a troubling question.
+      When in doubt, there might be someone you rely on for answers,
+      or you may rely on your upbringing's incomplete picture of things.
+    prompts:
+      - Violate unspoken rules or cultural taboos
+      - Cause conflict through misunderstanding
+      - Fall back on a trusted authority
+- front: &Noble
+    <<: *tfront
+    name: Noble
+    desc: >
+      //"Knights should be glorious champions of honor!" -- Emory
+
+      Your family might be Old Money,
+      or the most valuable thing you have is an old name.
+      Things are done just so, with elegance and refinement.
+      You hold a high station, and expectations of you
+      are equally high.
+      Otherwise, what is the point of you?
+
+      You've learned to approach situations through protocol.
+    prompts:
+      - Do what is right and proper, no matter what
+      - Hold others to a high standard
+      - Leverage your power and authority
+  back:
+    <<: *tback
+    name: Noble
+    desc: >
+      //"As opposed to ironclad thugs lording it over the peasants?" -- Basler
+
+      Not everyone benefits from, or approves of,
+      a system of aristocracy or nobility.
+      People might assume you look down on them,
+      or that you'll abuse the power you hold.
+      And all too often, they'll be right.
+    prompts:
+      - Mire a situation in pomp and etiquette
+      - Demand sacrifices you haven't made
+      - Lord it over your perceived lessers
+- front: &Nomadic
+    <<: *tfront
+    name: Nomadic
+    desc: >
+      //"The clear skies, the lure of the open road, it's perfect!" -- Tana
+
+      You grew up traveling, whether with family or other companions.
+      You've visited many communities, seen many interesting sights,
+      and heard many fascinating tales.
+      These experiences let you understand and embrace difference.
+
+      You've learned to approach situations through openness.
+    prompts:
+      - Apply a useful or relevant memory to a problem
+      - Bring a fresh perspective to a situation
+      - Forge a path to acceptance or compromise
```

```suggestion
      - Recall a story that will bring people together or solutions to light
      - Bring a fresh perspective to a situation
      - Forge a path to acceptance or compromise
```
The first one read as too vague to me. Just a suggestion.